content/blog/meet-our-new-hpe-dev-team-representative-at-hpe-technology-and-solution-summit-2023.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Meet our new HPE DEV Team Representative  at HPE Technology and Solution
+  Summit 2023!
+date: 2023-03-30T13:29:10.157Z
+author: Frederic Passeron
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+tags:
+  - HPE TSS
+  - HPE DEV
+---
+![](/img/tss2023.png "HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023")
+
+HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023 (HPE TSS 2023) goes physical again this year in sunny Barcelona. After several virtual editions, you will have the pleasure to engage with HPE Developer Community representative. All our sessions will be hold by our new team member : Mathieu Losmede. 
+
+H﻿PE TSS is the largest HPE technical event taking place in EMEA. Whether you are an HPE Presales or Partner, you will have the opportunity to attend technical breakout, Hands-on Labs, as well as strategic keynotes, and also finally get to meet your peers in person!
```

Comment:
Minor rewording suggestions:
"HPE TSS is the largest HPE technical event that is held in the Europe, Middle East, and Africa (EMEA) geography. HPE presales and partners have opportunities there to attend technical break-out sessions, hands-on labs, as well as strategic keynote presentations. They also get to enjoy meeting their peers in person!"